keps/sig-instrumentation/1661-event-api-migration/kep.yaml

```diff
@@ -0,0 +1,17 @@
+title: Event API Migration
+authors:
+  - "@chelseychen"
```

Comment:
you should add @gmarek to it - it's based on what he wrote

Reply:
Thanks for pointing this out. Added.